bin/dfx-canister-url

```diff
@@ -2,7 +2,6 @@
 set -euo pipefail
 SOURCE_DIR="$(dirname "$(realpath "${BASH_SOURCE[0]}")")"
 PATH="$SOURCE_DIR:$PATH"
-. "$SOURCE_DIR/versions.bash"
 
```

Comment:
Note: There are probably more files that import versions unnecessarily; I have not sought them all out.  I could do that for the next cleanup.  Started the PR but I'm out of time now.